ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,557 @@
+                      "type": "string"
+                    },
+                    "linkText": {
+                      "description": "Electronic access link text",
+                      "type": "string"
+                    },
+                    "publicNote": {
+                      "description": "Electronic access public note",
+                      "type": "string"
+                    },
+                    "relationshipId": {
+                      "description": "Electronic access relationship id",
+                      "type": "string",
+                      "$ref": "../uuid.json"
+                    },
+                    "materialsSpecification": {
+                      "description": "Electronic access materials specification",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "uri",
+                    "name",
+                    "linkText",
+                    "publicNote",
+                    "relationshipId",
+                    "materialsSpecification"
+                  ]
+                }
+              ]
+            },
+            "notes": {
+              "description": "Notes about action, copy, binding etc.",
+              "type": "array",
+              "items": [
+                {
+                  "holdingsNoteTypeName": {
+                    "description": "Name of the holdings note type",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Text content of the note",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "illPolicy": {
+              "description": "Name of the ILL policy",
+              "type": "string"
+            },
+            "retentionPolicy": {
+              "description": "Records information regarding how long we have agreed to keep something",
+              "type": "string"
+            },
+            "digitizationPolicy": {
+              "description": "Records information regarding digitization aspects",
+              "type": "string"
+            },
+            "holdingsStatements": {
+              "description": "Notes about action, copy, binding etc.",
+              "type": "array",
+              "items": [
+                {
+                  "holdingsNoteTypeName": {
+                    "description": "Name of the holdings note type",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Text content of the note",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "holdingsStatementsForIndexes": {
+              "description": "Holdings record indexes statements",
+              "type": "array",
+              "items": [
+                {
```

Comment:
This fails the build and should be changed to

              "items": {
                "type": "object",
                "properties": {

Reply:
You are right, it looks like I was wrong defined arrays of objects for some entries. I'll fix it, thanks.